src/test/java/org/jabref/logic/layout/format/LatexToUnicodeFormatterTest.java

```diff
@@ -110,6 +110,11 @@ void keepUnknownCommandWithoutArgument() {
         assertEquals("\\aaaa", formatter.format("\\aaaa"));
     }
 
+    @Test
+    void keepUnknownCommandWithNullArgument() {
+        assertEquals("", formatter.format(null));
```

Comment:
Good job on adding the test case, but since `null` isn't a valid argument for `format` I would assert a NullPointerException is thrown.